autogen/main/versions.tf.tmpl

```diff
@@ -24,33 +24,33 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = ">= 6.42.0, < 7"
+      version = ">= 6.47.0, < 7"
```

Comment:
Can you please update the README to match, and add an upgrade guide similar to https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/main/docs/upgrading_to_v38.0.md